projects/common-platform-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/messaging/Handler.kt

```diff
@@ -54,19 +54,44 @@ class Handler(
             }
```

Comment:
I know we haven't discussed this previously, but probably worth adding an event here too when we find a match - so we can see how many we get that already exist in Delius.